release.dockerfile

```
@@ -1,9 +1,30 @@
+COPY . .
+
+# Build arguments for version info
+ARG VERSION=dev
+ARG COMMIT=unknown
+ARG BUILD_DATE=unknown
+
+# Build the binary
+RUN CGO_ENABLED=0 go build -ldflags="-s -w -X main.version=${VERSION} -X main.commit=${COMMIT} -X main.date=${BUILD_DATE}" -o reportportal-mcp-server ./cmd/reportportal-mcp-server
+
+# Final stage
 FROM gcr.io/distroless/base-debian12
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)